.github/workflows/main.yml

```diff
@@ -73,30 +82,25 @@ jobs:
         run: |
           pip install --progress-bar off torch==${{ matrix.torch-version }}
           pip install --progress-bar off tensorflow==${{ matrix.tensorflow-version }}
-      - name: Install Texar
-        run: |
-          git clone https://github.com/asyml/texar-pytorch.git
-          cd texar-pytorch
-          pip install --use-feature=in-tree-build --progress-bar off .
-          cd ..
-          # Remove them to avoid confusion.
-          rm -rf texar-pytorch
       - name: Install Forte
         run: |
           pip install --use-feature=in-tree-build --progress-bar off .[models,test,wikipedia,data_aug,nlp,ir,texar-encoder,stave,audio_ext,remote,extractor]
+      - name: Test models module
+        run: |
+          pip install --use-feature=in-tree-build --progress-bar off .[test]
+          if [ ${{ matrix.modules.module }} != "" ]
+          then
+            pip install --use-feature=in-tree-build --progress-bar off .[${{ matrix.modules.module }}]
+            pip install --progress-bar off pytest==5.1.3 coverage
+          fi
       - name: Test backbone Forte import test
```

Comment:
`backbone Forte import` should be done before testing the `modules`, otherwise, some dependencies will be installed.